### PR TITLE
63 alert crud

### PR DESCRIPTION
**Pull request by pratyushpal**
Clean branch

### CODE REVIEW

File: controllers/Faq.js [hunk 1]
```diff
@@ -48,11 +47,11 @@ faqRouter.post('/insert', function(req, res) {
 });
 
 faqRouter.put('/edit', function(req, res) {
-	var questionToEdit = req.body;
+	const questionToEdit = req.body;
 
 	Faq.findById(questionToEdit.id, function(err, faq) {
-		if (err) return res.send(err);
-		else if (!faq) return res.send('Question ID not found!');
+		if (err) res.status(500).send(err);
```
Comment (theRoughCode) [inline controllers/Faq.js]: Why did you remove the `return` statement for both cases?

**Review by theRoughCode — CHANGES_REQUESTED** [review]
One comment

File: controllers/Faq.js (hunk 1 above)
Comment (pratyushpal) [reply controllers/Faq.js]: My bad. I think I messed up a bit while fixing conflicts

**Review by theRoughCode — APPROVED** [review]
